src/features/todo/todoSlice.jsx

```diff
@@ -0,0 +1,30 @@
+import { createSlice } from '@reduxjs/toolkit'
+
+const initialState = {
+    products: JSON.parse(localStorage.getItem('products')) || []
```

Comment:
let's remove all syncs with `localStorage` for now. it will only make things more complex. plus most of the times it won't be required in a real world application because all the data will be saved in the backend database and even if you used `localStorage` to cache them you'll also need a mechanism how to invalidate that cache. so let's just make things simpler and remove it for now.